spring-ai-alibaba-graph/spring-ai-alibaba-graph-core/pom.xml

```diff
@@ -166,6 +165,20 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>io.modelcontextprotocol.sdk</groupId>
+            <artifactId>mcp</artifactId>
+            <version>0.10.0</version>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>33.4.0-jre</version>
+            <optional>true</optional>
+        </dependency>
```

Comment:
Is this dependency necessary? It's very easy to cause conflict to user built applications, so better not use it.